.github/workflows/build_and_deploy.yml

```diff
@@ -12,12 +12,35 @@ on:
 
 jobs:
   call_build:
-    uses: yext/answers-search-ui/.github/workflows/build.yml@dev/build-workflow
+    uses: yext/answers-search-ui/.github/workflows/build.yml@feature/gh-action-migration
 
   call_unit_test:
-    uses: yext/answers-search-ui/.github/workflows/unit_test.yml@dev/build-workflow
+    uses: yext/answers-search-ui/.github/workflows/unit_test.yml@feature/gh-action-migration
     needs: call_build
 
+  format_branch_name:
+    runs-on: ubuntu-latest
+    outputs:
+      formatted_branch: ${{ steps.vars.outputs.formatted_branch }}
+    steps:
+      - name: Format branch name # replace '/' with '-'
+        id: vars
+        run: |
+          FORMATTED_BRANCH="$(echo ${GITHUB_REF_NAME} | sed "s/\//-/g")" 
+          echo $FORMATTED_BRANCH
+          echo ::set-output name=formatted_branch::${FORMATTED_BRANCH}
+
+  call_deploy:
+    needs: 
+      - call_unit_test
+      - format_branch_name
+    uses: yext/answers-search-ui/.github/workflows/deploy.yml@dev/gh-s3-deploy
```

Comment:
When GithubActins supports pointing to the HEAD ref, we'll update this: https://github.community/t/ref-head-in-reusable-workflows/203690

Reply:
Update: They just released the feature however users are reporting that it's not working yet

Reply:
2nd Update: The feature appears to be working now, so I updated the code!